deps/graph-parser/src/logseq/graph_parser/text.cljs

```diff
@@ -175,7 +175,7 @@
       v'
 
       (string/blank? v')
-      nil
+      ""
```

Comment:
Coulld we just delete this cond branch? The copy bug is still fixed and I still see that we have upstream code that trims blank space e.g.  `"     "`  -> `""`

Reply:
Deleted.